tests/tasks/test_two_moons.py

```diff
@@ -0,0 +1,193 @@
+import pytest
```

Comment:
(See comment for `test_norefposterior.py` above)

Reply:
Let's discuss this [here](https://github.com/sbi-benchmark/sbibm/pull/19#discussion_r743801750)